openai/manifest.json

```
@@ -0,0 +1,51 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "46e76528-8e93-4a7c-9299-387ce0a905c6",
+  "app_id": "openai",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Cost estimation, prompt sampling, performance metrics and more of OpenAI usage",
+    "title": "OpenAI",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
```

Just suggesting some classifier tags that are available in https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags!

```suggestion
      "Supported Data Type::Metrics",
      "Supported Data Type::Logs",
      "Supported Data Type::Traces",
      "Supported OS::Linux",
```